clients/identity/authentication.go

```diff
@@ -5,12 +5,21 @@ import (
 	"encoding/json"
 	"io/ioutil"
 	"net/http"
+	"strings"
 
 	"github.com/ONSdigital/go-ns/common"
 	"github.com/ONSdigital/go-ns/log"
 	"github.com/ONSdigital/go-ns/rchttp"
 )
 
+const prefix = "Bearer "
```

Comment:
You don't have to define this - it's already defined here: https://github.com/ONSdigital/go-ns/blob/4001bc2311e9cd2de98a365b6ff5ca6b1065e576/common/identity.go#L19